frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-utils.ts

```diff
@@ -149,6 +149,19 @@ export const getInitializationResource: AnnotationParser<K8sResourceKind> = (
     validate: isK8sResource,
     ...options,
   });
+export const getInitializationLink: AnnotationParser<URL> = (annotations, options) => {
+  const url = annotations?.[OLMAnnotation.InitializationLink];
+  if (!url) {
+    return null;
+  }
+
+  try {
+    return new URL(url);
```

Comment:
This causes an error. The CSV can only have a relative path. The base path needs to be appended by console. Without the base path it returns a invalid URL error.